view/frontend/web/js/data/translations-en_US.json

```diff
@@ -0,0 +1,55 @@
+{
+  "en_US": {
```

Comment:
Can we keep here flat structure without lang code `pl_PL `?

Reply:
no problem